### PR TITLE
更新 Lightning 所需权限描述

### PR DESCRIPTION
**Pull request by sunzhaoyang**
且只留一个地方，多余的删掉

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — COMMENTED** [review]
LGTM

File: tidb-lightning/tidb-lightning-requirements.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+## Lightning 运行时资源要求
+
+**操作系统**：一般建议使用新的、纯净版 CentOS 7 实例，您可以在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程默认会占满 CPU，建议单独部署。如果条件不允许，可以和其他组件（比如 `tikv-server`）部署在同一台机器上，然后通过配置 `region-concurrency` 限制 TiDB Lightning 使用 CPU 资源。混合部署的情况下可以将其大小配置为逻辑 CPU 数的 75%，以限制 CPU 的使用。
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**存储空间**：配置项 `sorted-kv-dir` 设置排序的键值对的临时存放地址，目标路径需要是一个空目录，至少需要数据源最大单表的空间。建议与 `data-source-dir` 使用不同的存储设备，独占 IO 会获得更好的导入性能，且建议优先考虑配置闪存等高性能存储介质。
+
+**网络**： 建议使用 10Gb 网卡。
```
Comment (glorv) [inline tidb-lightning/tidb-lightning-requirements.md:113]:
```suggestion
**网络**： 建议使用带宽 >=10Gbps 的网卡。
```

**Review by hfxsd — APPROVED** [review]

File: tidb-lightning/tidb-lightning-requirements.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
+---
+title: 使用 TiDB Lightning 所需要的前提条件
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:2]:
```suggestion
title: 使用 TiDB Lightning 所需要的前提条件
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+summary: 了解 TiDB Lightning 运行时所需要的各种要求。
+---
+
+# 使用 Lightning 时所需要的前提条件
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:6]: 目前标题比较长，要不要改成 “TiDB Lightning 使用前提”

File: tidb-lightning/tidb-lightning-requirements.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+使用 Lightning 导入数据之前，预先检查环境确认是否满足要求，有助于减少导入过程的错误，避免导入失败的情况。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:8]:
```suggestion
使用 Lightning 导入数据前，先检查环境是否满足要求，这有助于减少导入过程的错误，避免导入失败的情况。
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+## 下游数据库权限要求
+
+Lightning 导入数据时，根据所使用的导入方式，启用的特性等，需要下游数据库用户具备不同的权限，可参考下表：
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:12]:
```suggestion
Lightning 导入数据时，根据导入方式和启用特性等，需要下游数据库用户具备不同的权限，可参考下表：
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+      <td>-</td>
+      <td>RESTRICTED_VARIABLES_ADMIN,RESTRICTED_TABLES_ADMIN</td>
+      <td>当目标 TiDB 开启 SEM</td>
+   </tr>
+   <tr>
+      <td>推荐</td>
+      <td>冲突检测，max-error</td>
+      <td>lightning.task-info-schema-name 配置的 schema</td>
+      <td>SELECT,INSERT,UPDATE,DELETE,CREATE,DROP</td>
+      <td>如不需要，该值必需设为""</td>
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:62]:
```suggestion
      <td>如不需要，该值必须设为""</td>
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+   </tr>
+   <tr>
+      <td>可选</td>
+      <td>并行导入</td>
+      <td>lightning.meta-schema-name 配置的 schema</td>
+      <td>SELECT,INSERT,UPDATE,DELETE,CREATE,DROP</td>
+      <td>如不需要，该值必需设为""</td>
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:69]:
```suggestion
      <td>如不需要，该值必须设为""</td>
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+   </tr>
+   <tr>
+      <td>可选</td>
+      <td>checkpoint.driver = “mysql”</td>
+      <td>checkpoint.schema 设置</td>
+      <td>SELECT,INSERT,UPDATE,DELETE,CREATE,DROP</td>
+      <td>使用数据库而非文件形式存放 checkpoint 信息时</td>
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:76]:
```suggestion
      <td>使用数据库而非文件形式存放 checkpoint 信息时需要</td>
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+
+{{< copyable "sql" >}}
+
+```sql
+select table_name,table_schema,sum(data_length)/1024/1024 as data_length,sum(index_length)/1024/1024 as index_length,sum(data_length+index_length)/1024/1024 as sum from information_schema.tables where table_schema = "${schema_name}" group by table_name,table_schema order by sum  desc limit 5;
+```
+
+## Lightning 运行时资源要求
+
+**操作系统**：一般建议使用新的、纯净版 CentOS 7 实例，您可以在本地虚拟化或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程默认会占满 CPU，建议单独部署。如果条件不允许，可以和其他组件（比如 `tikv-server`）部署在同一台机器上，然后通过配置 `region-concurrency` 限制 TiDB Lightning 使用 CPU 资源。混合部署的情况下可以将其大小配置为逻辑 CPU 数的 75%，以限制 CPU 的使用。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:107]: 在[从大数据量 MySQL 迁移数据到 TiDB](https://docs.pingcap.com/zh/tidb/dev/migrate-large-mysql-to-tidb/#%E8%B5%84%E6%BA%90%E8%A6%81%E6%B1%82)这个文档，这部分信息是这么描述的，看看要不要保持一致：

本文档示例使用的是若干新的、纯净版 CentOS 7 实例，你可以在本地虚拟化一台主机，或在供应商提供的平台上部署一台小型的云虚拟主机。TiDB Lightning 运行过程中，默认会占满 CPU，建议单独部署在一台主机上。如果条件不允许，你可以将 TiDB Lightning 和其他组件（比如 `tikv-server`）部署在同一台机器上，然后设置 `region-concurrency` 配置项的值为逻辑 CPU 数的 75%，以限制 TiDB Lightning 对 CPU 资源的使用。

File: tidb-lightning/tidb-lightning-requirements.md [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+**内存和 CPU**：因为 TiDB Lightning 对计算机资源消耗较高，建议分配 64 GB 以上的内存以及 32 核以上的 CPU，而且确保 CPU 核数和内存（GB）比为 1:2 以上，以获取最佳性能。
+
+**存储空间**：配置项 `sorted-kv-dir` 设置排序的键值对的临时存放地址，目标路径需要是一个空目录，至少需要数据源最大单表的空间。建议与 `data-source-dir` 使用不同的存储设备，独占 IO 会获得更好的导入性能，且建议优先考虑配置闪存等高性能存储介质。
+
+**网络**： 建议使用带宽 >=10Gbps 的网卡。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:113]:
```suggestion
**网络**：建议使用带宽 >=10Gbps 的网卡。
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 11]
```diff
@@ -27,44 +27,18 @@ summary: 使用 Dumpling 和 TiDB Lightning 合并导入分表数据到 TiDB，
 
 - [使用 TiUP 安装 DM 集群](/dm/deploy-a-dm-cluster-using-tiup.md)
 - [使用 TiUP 安装 Dumpling 和 Lightning](/migration-tools.md)
-- [DM 所需上下游数据库权限](/dm/dm-worker-intro.md)
-- [Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-faq.md#tidb-lightning-对下游数据库的账号权限要求是怎样的)
 - [Dumpling 所需上游数据库权限](/dumpling-overview.md#从-tidbmysql-导出数据)
+- [Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-requirements.md#下游数据库权限要求)
+- [下游数据库所需空间](/tidb-lightning/tidb-lightning-requirements.md#下游数据库所需空间)
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:32]:
```suggestion
- [Lightning 下游数据库所需空间](/tidb-lightning/tidb-lightning-requirements.md#下游数据库所需空间)
```
跟上面的 “Lightning 所需下游数据库权限”保持风格一致。

File: migrate-large-mysql-shards-to-tidb.md [hunk 12]
```diff
@@ -27,44 +27,18 @@ summary: 使用 Dumpling 和 TiDB Lightning 合并导入分表数据到 TiDB，
...
-select table_name,table_schema,sum(data_length)/1024/1024 as data_length,sum(index_length)/1024/1024 as index_length,sum(data_length+index_length)/1024/1024 as sum from information_schema.tables where table_schema = "${schema_name}" group by table_name,table_schema order by sum  desc limit 5;
-```
-
-### 目标 TiKV 集群的磁盘空间要求
-
-**磁盘空间**：目标 TiKV 集群必须有足够空间接收新导入的数据。除了[标准硬件配置](/hardware-and-software-requirements.md)以外，目标 TiKV 集群的总存储空间必须大于 **数据源大小 × [副本数量](/faq/deploy-and-maintain-faq.md#每个-region-的-replica-数量可配置吗调整的方法是) × 2**。例如集群默认使用 3 副本，那么总存储空间需为数据源大小的 6 倍以上。公式中的 2 倍可能难以理解，其依据是以下因素的估算空间占用：
-
-* 索引会占据额外的空间
-* RocksDB 的空间放大效应
+关于 Lightning 运行时所需资源，全量文件空间估算，及导入时下游数据库所需空间请参考 [Lightning 所需前提条件](/tidb-lightning/tidb-lightning-requirements.md)
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:37]: 上面已经有所需空间的链接了，这里第 35-38 行删除？

File: tidb-lightning/tidb-lightning-faq.md [hunk 13]
```diff
@@ -17,22 +17,7 @@ TiDB Lightning 的版本应与集群相同。如果使用 Local-backend 模式
...
-
-如果选择 [TiDB-backend](/tidb-lightning/tidb-lightning-backends.md#tidb-lightning-tidb-backend) 模式，或目标数据库用于存储断点，则 TiDB Lightning 额外需要以下权限：
-
-* INSERT
-* DELETE
-
-Local-backend 和 Importer-backend 无需以上两个权限，因为数据直接被 Ingest 到 TiKV 中，所以绕过了 TiDB 的权限系统。只要 TiKV、TiKV Importer 和 TiDB Lightning 的端口在集群之外不可访问，就可以保证安全。
-
-如果 TiDB Lightning 配置项 `checksum = true`，则 TiDB Lightning 需要有下游 TiDB admin 用户权限。
+详细权限描述参考 [使用 Lightning 时所需要的前提条件](/tidb-lightning/tidb-lightning-requirements.md)
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-faq.md:20]:
```suggestion
详细权限描述参考 [TiDB Lightning 使用前提](/tidb-lightning/tidb-lightning-requirements.md)。
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 14]
```diff
@@ -0,0 +1,113 @@
+---
+title: TiDB Lightning 使用前提
+summary: 了解 TiDB Lightning 运行时所需要的各种要求。
+---
+
+# 使用 Lightning 时所需要的前提条件
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:6]:
```suggestion
# TiDB Lightning 使用前提
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 15]
```diff
@@ -0,0 +1,113 @@
...
+
+使用 Lightning 导入数据前，先检查环境是否满足要求，这有助于减少导入过程的错误，避免导入失败的情况。
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:8]:
```suggestion
使用 TiDB Lightning 导入数据前，先检查环境是否满足要求，这有助于减少导入过程的错误，避免导入失败的情况。
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 16]
```diff
@@ -0,0 +1,113 @@
...
+
+## 下游数据库权限要求
+
+Lightning 导入数据时，根据导入方式和启用特性等，需要下游数据库用户具备不同的权限，可参考下表：
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:12]:
```suggestion
TiDB Lightning 导入数据时，根据导入方式和启用特性等，需要下游数据库用户具备不同的权限，可参考下表：
```

File: tidb-lightning/tidb-lightning-requirements.md [hunk 17]
```diff
@@ -0,0 +1,113 @@
...
+
+统计最大单表，单位 MiB，注意修改 ${schema_name}
+
+{{< copyable "sql" >}}
+
+```sql
+select table_name,table_schema,sum(data_length)/1024/1024 as data_length,sum(index_length)/1024/1024 as index_length,sum(data_length+index_length)/1024/1024 as sum from information_schema.tables where table_schema = "${schema_name}" group by table_name,table_schema order by sum  desc limit 5;
+```
+
+## Lightning 运行时资源要求
```
Comment (shichun-0415) [inline tidb-lightning/tidb-lightning-requirements.md:105]:
```suggestion
## TiDB Lightning 运行时资源要求
```

File: migrate-large-mysql-shards-to-tidb.md [hunk 18]
```diff
@@ -27,44 +27,18 @@ summary: 使用 Dumpling 和 TiDB Lightning 合并导入分表数据到 TiDB，
 
 - [使用 TiUP 安装 DM 集群](/dm/deploy-a-dm-cluster-using-tiup.md)
 - [使用 TiUP 安装 Dumpling 和 Lightning](/migration-tools.md)
-- [DM 所需上下游数据库权限](/dm/dm-worker-intro.md)
-- [Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-faq.md#tidb-lightning-对下游数据库的账号权限要求是怎样的)
 - [Dumpling 所需上游数据库权限](/dumpling-overview.md#从-tidbmysql-导出数据)
+- [Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-requirements.md#下游数据库权限要求)
```
Comment (shichun-0415) [inline migrate-large-mysql-shards-to-tidb.md:31]:
```suggestion
- [TiDB Lightning 所需下游数据库权限](/tidb-lightning/tidb-lightning-requirements.md#下游数据库权限要求)
```
如可以，尽量统一用 TiDB Lightning，避免术语不一致。

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**